css/types/color.json

```diff
@@ -1027,6 +1027,44 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "mark_marktext_buttonborder": {
```

Comment:
@queengooborg I did this as a subfeature combining all three system colors. I can split them up if you want. The reason I did them combined is that they were added to the spec at the same time and both FF and Chrome had them in grouped bugs. So I would expect them to always deliver together. 

IF that proves incorrect we can split them later. Or if you want we can split now?

Reply:
I think that grouping is fine if these three colors are expected to always be implemented at the same time, and we can separate them later if we need to!